indykite_sdk/model/identity_knowledge.py

```diff
@@ -74,52 +92,53 @@ def deserialize(cls, relationship):
             source=relationship.source,
             target=relationship.target,
         )
+        if relationship.create_time:
+            ik_relationship.create_time = timestamp_to_date(relationship.create_time)
+        if relationship.update_time:
+            ik_relationship.update_time = timestamp_to_date(relationship.update_time)
         property_map = {
             k: Property(arg_to_value(v))
             for k, v in relationship.properties.items()
         }
         ik_relationship.properties = property_map
         return ik_relationship
 
-    def __init__(self, id=None, type=None, source=None, target=None, properties=None):
+    def __init__(self, id=None, type=None, source=None, target=None, create_time=None, update_time=None,
+                 properties=None):
         self.id = id
         self.type = type
         self.source = source
         self.target = target
+        self.create_time = create_time
+        self.update_time = update_time
         self.properties = properties
 
 
-class Path:
-    @classmethod
-    def deserialize(cls, path):
-        if path is None:
-            return None
-        ik_path = Path()
-        nodes = []
-        for node in path.nodes:
-            nodes.append(Node.deserialize(node))
-        ik_path.nodes = nodes
-        relationships = []
-        for relationship in path.relationships:
-            relationships.append(Relationship.deserialize(relationship))
-        ik_path.relationships = relationships
-        return ik_path
-
-    def __init__(self, nodes=[], relationships=[]):
-        self.nodes = nodes
-        self.relationships = relationships
-
-
 class Property:
     @classmethod
     def deserialize(cls, property):
         if property is None:
             return None
         return Property(
-            property.key if property.key else None,
+            property.type if property.type else None,
             object_to_value(property.value) if property.value else None
         )
 
-    def __init__(self, key=None, value=None):
-        self.key = key
+    def __init__(self, type=None, value=None):
+        self.type = type
         self.value = value
+
+
+class Return:
+    @classmethod
+    def deserialize(cls, returned):
+        if returned is None:
+            return None
+        return Return(
+            returned.variable if returned.variable else None,
+            returned.properties if returned.properties else None,
+        )
```

Comment:
You can use "or" to make it more readable
```suggestion
        return cls(
            returned.variable or None,
            returned.properties or None
        )
```